org.tribler.Tribler.yaml

```diff
@@ -43,13 +45,13 @@ modules:
       - type: script
         dest-filename: tribler
         commands:
-          - exec /app/share/tribler/tribler "$@"
+          - export TMPDIR="${XDG_CACHE_HOME}"/tmp/
+          - exec /app/share/share/tribler/tribler "$@"
```

Comment:
This line looks a bit strange to me, sure that it's `/app/share/share/tribler/`?

Reply:
That's what worked on my machine. I think there are differences between my local build and the test build.